app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -165,85 +169,94 @@ const RentalAssistanceForm = ({ values, onSave, loading, onClose, applicationMem
   return (
     <>
       <div className={classNames(
+        // Convert these to IDs, they are only being used for tests.
         {
           'rental-assistance-new-form': isNew,
           'rental-assistance-edit-form': !isNew
         }
       )}>
-        <FormGrid.Row expand={false}>
-          <FormGrid.Item>
-            <SelectField
-              label='Type of Assistance'
-              fieldName={`rental_assistances.${index}.type_of_assistance`}
-              options={typeOfAssistanceOptions}
-              className='rental-assistance-type'
-              validation={validate.isPresent('Please select a type of assistance.')}
-            />
-          </FormGrid.Item>
-          <FormGrid.Item>
-            <YesNoRadioGroup
-              label='Recurring Assistance'
-              fieldName={`rental_assistances.${index}.recurring_assistance`}
-              uniqId={(!isEmpty(values) && values.id) || 'new'}
-              trueValue='Yes'
-              falseValue='No'
-              className='rental-assistance-recurring'
-            />
-          </FormGrid.Item>
-          <FormGrid.Item>
-            <CurrencyField
-              label='Assistance Amount'
-              fieldName={`rental_assistances.${index}.assistance_amount`}
-              validation={validateAssistanceAmount}
-              id='assistance_amount'
-              isDirty={visited && visited[`rental_assistances.${index}.assistance_amount`]}
-            />
-          </FormGrid.Item>
-          <FormGrid.Item>
-            <SelectField
-              label='Recipient'
-              fieldName={`rental_assistances.${index}.recipient`}
-              options={applicationMembersOptions}
-              className='rental-assistance-recipient'
-            />
-          </FormGrid.Item>
-        </FormGrid.Row>
-        {isOther(form.getState().values.rental_assistances[index]) && (
-          <FormGrid.Row expand={false}>
+        <InlineModal whiteBackground marginBottom={isNew} >
+          <FormGrid.Row>
+            <FormGrid.Item>
+              <SelectField
+                label='Recipient'
+                fieldName={`rental_assistances.${index}.recipient`}
+                options={applicationMembersOptions}
+                className='rental-assistance-recipient'
+              />
+            </FormGrid.Item>
+          </FormGrid.Row>
+          <FormGrid.Row>
             <FormGrid.Item>
-              <InputField
-                label='Other Assistance Name'
-                fieldName={`rental_assistances.${index}.other_assistance_name`} />
+              <SelectField
+                label='Type of Assistance'
+                fieldName={`rental_assistances.${index}.type_of_assistance`}
```

Comment:
I'm seeing `rental_assistances.${index}.*` a lot. I wonder if this would be better pulled out as a simple function like:

```
const getField = (assistanceField) => `rental_assistances.${index}.${assistanceField}`
```

and then this line could be `fieldName={getField('type_of_assistance')}`